src/main/java/ch/njol/skript/registrations/Feature.java

```diff
@@ -11,13 +12,177 @@
  * Experimental feature toggles as provided by Skript itself.
  */
 public enum Feature implements Experiment {
-	EXAMPLES("examples", LifeCycle.STABLE),
-	QUEUES("queues", LifeCycle.EXPERIMENTAL),
-	FOR_EACH_LOOPS("for loop", LifeCycle.EXPERIMENTAL, "for [each] loop[s]"),
-	SCRIPT_REFLECTION("reflection", LifeCycle.EXPERIMENTAL, "[script] reflection"),
-	CATCH_ERRORS("catch runtime errors", LifeCycle.EXPERIMENTAL, "error catching [section]"),
-	TYPE_HINTS("type hints", LifeCycle.EXPERIMENTAL, "[local variable] type hints"),
-	DAMAGE_SOURCE("damage source", LifeCycle.EXPERIMENTAL, "damage source[s]")
+
+	EXAMPLES("examples", LifeCycle.STABLE) {
+		@Override
+		public @NotNull String displayName() {
+			return "Examples";
+		}
+
+		@Override
+		public @NotNull String description() {
+			return """
+				A section used to provide examples inside code.
+				""";
+		}
+	},
```

Comment:
this is a pretty clunky way to do this. Surely we could do it through parameters, or not have Feature be an enum?